### PR TITLE
SDK-80 [Node.js SDK] Add file upload/read methods

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-80](https://incountry.atlassian.net/browse/SDK-80)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# InCountry Storage SDK
+# InCountry Storage SDK [File Storage DRAFT]
```
[the remark this answers is not in the document]
Comment (igor-alekseev-dev) [reply README.md:1]: @Rommero said if we don't have this comment QAs won't accept this docs %)

**Review by artemgolozubov — DISMISSED** [review]
LGTM

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Pull request merged**